src/main/java/lotto/domain/Lotto.java

```diff
@@ -30,61 +30,77 @@ private Lotto(List<LottoNumber> numbers) {
     }
 
     public static Lotto from(List<LottoNumber> number) {
-        return new Lotto(number);
+        return new Lotto(number.stream()
+                .collect(Collectors.toSet()));
     }
 
     public static Lotto winningFrom(List<Integer> number) {
 
-        List<LottoNumber> numbers = new ArrayList<>();
-        for(int i = 0; i < number.size(); i++) {
-            numbers.add(new LottoNumber(number.get(i)));
+        Set<LottoNumber> numbers = new TreeSet<>();
+        Iterator<Integer> itr = number.iterator();
+        while(itr.hasNext()) {
+            numbers.add(new LottoNumber(itr.next()));
         }
```

Comment:
```suggestion
        for (Integer integer : number) {
            numbers.add(new LottoNumber(integer));
        }
```
향상된 for문을 사용하면 내부적으로 iterator를 사용해서 바로 내부 값을 순회할 수 있습니다. 코드를 더 단축시킬 수 있죠. 

그리고 여기서 Stream API를 쓸 수도 있겠죠. ㅎ(이건 취향입니다. )